package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "lodash.assign": "^4.2.0",
     "lodash.intersection": "^4.4.0",
+    "lodash.isplainobject": "^4.0.6",
```

Comment:
I think all these specialized lodash repos are just wrapping the normal lodash repos, and then just export one function. So make sure you are not loading the different versions of lodash here. Given that you use so many lodash repos, maybe just load lodash by itself?